pkg/daemon/daemon.go

```diff
@@ -56,6 +56,8 @@ type Daemon struct {
 
 	// OperatingSystem the operating system the MCD is running on
 	OperatingSystem string
+	// isOstree is true if OperatingSystem uses rpm-ostree
```

Comment:
Rather than adding a new field, why not just give `OperatingSystem` its own type with an `isOstree` method? Written as is, there is the possibility to encode an invalid state into this object.

Reply:
Good suggestion, thanks!  I did a larger rework along those lines.